packages/teams-js/src/public/meeting.ts

```diff
@@ -560,4 +586,60 @@ export namespace meeting {
       sendMessageToParent('meeting.appShareButton.setOptions', [shareInformation]);
     }
   }
+
+  /**
+   * Ext App will handle the Audio(Mic & Speaker). Teams to go Audio less (No Audio)
+   *
+   * @throws standard parameter validation error
+   * @param isAppHandlingAudio - true - expect App to handle the audio and Teams to go audioless.
+   *    false - Audio transferred to Teams and Teams to restore the audio.
+   * @param callbackMicMuteStateChangedHandler - Callback contains mic status param.
+   * @param callback - Callback contains IRequestAppAudioHandlingSdkResponse param, error or operation result.
+   * error can either contain an error of type SdkError (error indication), or null (non-error indication)
+   * result can either contain a true boolean value (successful termination), or null (unsuccessful fetch)
+   * @beta
+   */
+  export function requestAppAudioHandling(
+    callback: (
+      error: SdkError | null,
+      requestAppAudioHandlingSdkResponse: IRequestAppAudioHandlingSdkResponse | null,
+    ) => void,
+    isAppHandlingAudio: boolean,
+    callbackMicMuteStateChangedHandler: (micState: IMicState) => void,
```

Comment:
Can you please combine the isAppHandlingAudio and callbackMicMuteStateChangedHandler into a RequestAppAudioHandlingParams object so that it's easy to add more properties in the future. Also, move this parameter ahead of the callback (which should come last).